github_integration/provider.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    apstra = {
+      source = "Juniper/apstra"
+    }
```

Comment:
We should probably version lock the provider here so it doesn't break in the future.